.github/ISSUE_TEMPLATE/usability.md

```diff
@@ -0,0 +1,26 @@
+---
+name: Usability Improvement
+about: Make Bevy more fun to use!
+title: ''
+labels: C-Usability, S-Needs-Triage
+assignees: ''
+---
+
+## What tripped you up?
+
+Was something inconsistent, surprising or frustrating?
+Try to provide specific examples if you can.
+These could be code snippets, screenshots, a link to a bad error message in our source code, etc.
+
+If this is part of a broader problem, feel free to point out other areas where this needs to be fixed.
```

Comment:
I kind of want to cut this line. If someone thinks their issue is part of a broader problem, I think we can assume they will call that out here.

(sorry for continually trying to cut things out of your templates. I just really want to keep these short and sweet)